test/module/irohad/consensus/yac/CMakeLists.txt

```diff
@@ -20,27 +20,31 @@ addtest(cluster_order_test cluster_order_test.cpp)
 target_link_libraries(cluster_order_test
     yac
     shared_model_cryptography_model
+    shared_model_stateless_validation
```

Comment:
Why is stateless validation required? I expected that builders required but not a validation.